setup.cfg

```diff
@@ -27,25 +27,24 @@ packages = find:
 python_requires = >=3.8
 zip_safe = False
 install_requires =
-    invenio-oaiserver>=2.2.0,<3.0.0
-    invenio-requests>=4.0.0,<5.0.0
-    invenio-search-ui>=2.4.0,<3.0.0
-    invenio-vocabularies>=3.0.0,<4.0.0
-    invenio-administration>=2.0.0,<3.0.0
+    invenio-oaiserver~=2.4.0
+    invenio-requests~=4.1.3
```

Comment:
do we want this? does this mean that we can't release invenio-requests as minor because it wouldn't be picked up to app-rdm because of this ~= operator?

Reply:
at least i understand the operator like that, [reference](https://peps.python.org/pep-0440/#compatible-release)